dom/src/mockDOMSource.ts

```diff
@@ -1,81 +1,81 @@
 import {StreamAdapter, DevToolEnabledSource} from '@cycle/base';
+import {DOMSourceable, GenericStream, EventsFnOptions} from './DOMSourceable';
 import {VNode} from './interfaces';
-import xsSA from '@cycle/xstream-adapter';
-import {DOMSource, EventsFnOptions} from './DOMSource';
+import xsAdapter from '@cycle/xstream-adapter';
 import xs from 'xstream';
 
-export type GenericStream = any;
-export type ElementStream = any;
-export type EventStream = any;
-
 export type MockConfig = {
   [name: string]: GenericStream | MockConfig;
   elements?: GenericStream;
 }
 
-const SCOPE_PREFIX = '___';
+const CYCLE_SOURCE = `MockedDOM`;
+const CSS_NOTATION = `.`;
+const SCOPE_PREFIX = `___`;
 
-export class MockedDOMSource implements DOMSource {
+export class MockedDOMSource implements DOMSourceable {
   private _elements: any;
 
   constructor(private _streamAdapter: StreamAdapter,
               private _mockConfig: MockConfig) {
-    if (_mockConfig.elements) {
-      this._elements = _mockConfig.elements;
-    } else {
-      this._elements = _streamAdapter.adapt(xs.empty(), xsSA.streamSubscribe);
-    }
+    this._elements = _mockConfig.elements ||
+                     _streamAdapter.adapt(xs.empty(), xsAdapter.streamSubscribe);
```

Comment:
We don't use this type of indentation/alignment anywhere in the codebase, and while it's more aesthetic, I find it rather harmful, because if we rename `this._elements` to a name that is shorter or longer, we need to reindent the second line, and that will show up in the git diff.